### DISCUSSION TITLE
POST request return messages

### DISCUSSION THREAD

**Discussion by Tooloco** [firefly-iii/firefly-iii#6704] [Firefly III]
Hello,

Im developing a script that POSTs transaction  data to my firefly importer instance. I'm trying to find out how to detect if the submission has been succesful or not. It seems the response code is the same for both (200). The docs dont mention much about POST importing (https://docs.firefly-iii.org/data-importer/usage/post/)

If anyone could lend me a helping hand and indicate what kind of messages you get back when succesfully importing, unsuccesful importing / duplicate detection... etc.

Thanks!

**Comment by JC5** [comment]
At the moment, you don't. The result can be observed from the logs and/or from the error message that the Data Importer mails.
